iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -553,7 +573,11 @@ private void commitOverwrite(Table table, String branchName, long startTime, Fil
       LOG.info("Cleared table contents as part of empty overwrite for unpartitioned table. " +
           "Commit took {} ms for table: {}", System.currentTimeMillis() - startTime, table);
     }
-
+    if (rewritePolicy != RewritePolicy.DEFAULT) {
+      LOG.info("Completed compaction for table {}, number of files committed: {}",
```

Comment:
That's not generic - 'compaction'. That msg should be logged on the QueryCompactor side

Reply:
Done